src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -70,3 +70,174 @@ describe('List rendering', () => {
     });
   });
 });
+
+describe('Dataset list filters', () => {
+  beforeEach(function () {
```

Comment:
No, this is needed because of the 'this' value.

But searching for the correct Cypress docs for this response. I see that you can now avoid using 'this' with an alias, so I will use that.

https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Avoiding-the-use-of-this

Reply:
Nice!

Reply:
This does add an extra nested layer. Because you need .then() to get the value. I still prefer the alias version above using `this`.

Note: [.then()](https://docs.cypress.io/api/commands/then) is a Cypress command, not a Promise. This means you cannot use things like async/await within your Cypress tests.